src/DevLicenseDimo.sol

```diff
@@ -107,28 +108,14 @@ contract DevLicenseDimo is DevLicenseMeta {
      * @param enabled True to enable the URI, false to disable it.
      * @param uri The redirect URI to modify.
      */
-    function setRedirectUri(
-            uint256 tokenId, 
-            bool enabled, 
-            string calldata uri
-        ) onlyTokenOwner(tokenId) external {
-            if(enabled) {
-                emit RedirectUriEnabled(tokenId, uri);
-            } else {
-                emit RedirectUriDisabled(tokenId, uri);
-            }
-        _redirectUris[tokenId][uri] = enabled;
-    }
-
-    /**
-     * @notice Removes a redirect URI for a specific token.
-     * @dev Only the token owner can call this function.
-     * @param tokenId The ID of the token.
-     * @param uri The URI to remove.
-     */
-    function removeRedirectUri(uint256 tokenId, string calldata uri) onlyTokenOwner(tokenId) external {
-        delete _redirectUris[tokenId][uri];
-        emit RedirectUriDisabled(tokenId, uri);
+    function setRedirectUri(uint256 tokenId, bool enabled, string calldata uri) external onlyTokenOwner(tokenId) {
+        if (enabled) {
+            _redirectUris[tokenId][uri] = enabled;
```

Comment:
What if this is already true? Is this still better for gas? Emitting the event seems strange, too; although I know that the major token contracts allow self-transfer. Maybe this is like that.